b9lab-content/3-main-concepts/13-events.md

```diff
@@ -61,35 +63,59 @@ func NewHandler(keeper Keeper) sdk.Handler {
         switch msg := msg.(type) {
     // event types
 ```
-</HighlightBox>
 
+</HighlightBox>
 
 ## Subscribing to Events
 
-You can use Tendermint's Websocket (opens new window) to subscribe to Events by calling the subscribe RPC method.
+You can use Tendermint's Websocket (opens new window) to subscribe to events by calling the subscribe RPC method.
 
-The main eventCategories you can subscribe to are:
+The main `eventCategories` you can subscribe to are:
 
-* **NewBlock**: Contains Events triggered during BeginBlock and EndBlock.
-* **Tx**: Contains Events triggered during DeliverTx (i.e. transaction processing).
+* **NewBlock**: Contains Events triggered during `BeginBlock` and `EndBlock`.
+* **Tx**: Contains events triggered during `DeliverTx` (i.e. transaction processing).
 * **ValidatorSetUpdates**: Contains validator set updates for the block.
 
 <HighlightBox type=”info”>
-→ full list of event categories at: https://godoc.org/github.com/tendermint/tendermint/types#pkg-constants
-</HighlightBox>
-
-You can filter for event types and attribute values.For example, a transfer transaction triggers an Event of type Transfer and has Recipient and Sender as attributes (as defined in the events.go file of the bank module
 
-## Long-running exercise
+→ full list of event categories [here](https://godoc.org/github.com/tendermint/tendermint/types#pkg-constants)
 
-We want to surface some information that is usable for server systems and GUIs:
+</HighlightBox>
 
-* Per transaction:
-    * The game was created. Both players need to be notified.
-    * The player has played. The other player needs to be notified.
-    * The game was rejected. The other player needs to be notified.
-    * Perhaps the game was won. Both players need to be notified.
-* Per block:
-    * These games timed out. The game ids and the involved players need to be notified.
+You can filter for event types and attribute values. For example, a transfer transaction triggers an event of type `Transfer` and has `Recipient` and `Sender` as attributes (as defined in the `events.go` file of the bank module.
+
```

Comment:
Not a priority, but a wrap up and CTA would be desirable.

Reply:
Updated.